### ISSUE TITLE
ghidra-stubs PyPI package

### ISSUE THREAD

**Issue by tmr232** [NationalSecurityAgency/ghidra#7317]
Hello!

I'm the maintainer of the [PyPI `ghidra-stubs` package](https://pypi.org/project/ghidra-stubs/).
Since Ghidra is now generating 1st-party Python stubs, I figured it'd make sense to give you control of the PyPI package so that you can publish the 1st-party stubs instead of my [3rd-party ones](https://github.com/VDOO-Connected-Trust/ghidra-pyi-generator).

I think it would benefit the users of Ghidra and avoid confusion.

If you're interested, please let me know.

**Comment by ryanmkurtz** [comment]
Yes, very interested.  I planned on reaching out to you very soon for the same request.  I'll DM you.

**Comment by ReversingWithMe** [comment]
Not to intentionally necro this

but related to this:
https://github.com/ReversingWithMe/GhidraSnippets?tab=readme-ov-file#scripting-and-plugins
```
One gotcha is the location of the .pyi files is 1 level removed, so this can be corrected by making 
the expected structure and reorganizing the files into that new file. 
So instead of looking for util.XYZ, it will be ghidra.util.XYZ

    mkdir venv/lib/python3.12/site-packages/ghidra-stubs/ghidra
    mv venv/lib/python3.12/site-packages/ghidra-stubs/* ./venv/lib/python3.12/site-packages/ghidra-stubs/ghidra/
```

Any idea if I am using the stubs wrong, or what are the odds the stubs could be refactored that way?

<img width="577" height="414" alt="Image" src="https://github.com/user-attachments/assets/c414d1ce-42bb-4b50-8aff-0f3dcf0c0e77" />

I found that I had to move the stubs down a directory for a nice ergonomic usage in my IDEs, but I am not that used to using stubs personally.

**Comment by ryanmkurtz** [comment]
@ReversingWithMe I just went through these steps and had no issue in VSCode when working with Ghidra's `Msg` class:

1. `python3 -m venv venv`
2. `source venv/bin/activate`
3. `pip3 install pyghidra ghidra_stubs`

<img width="881" height="379" alt="Image" src="https://github.com/user-attachments/assets/7279901f-874e-4b91-9186-3ada9c26c46b" />

<img width="663" height="431" alt="Image" src="https://github.com/user-attachments/assets/3acacba2-5642-4a00-8a77-4fb72629141f" />

**Comment by ReversingWithMe** [comment]
Understood, thanks!
